platform/darwin/src/MGLLoggingConfiguration.m

```diff
@@ -42,21 +42,66 @@ - (MGLLoggingBlockHandler)defaultBlockHandler {
         
         if (@available(iOS 10.0, macOS 10.12.0, *)) {
             static dispatch_once_t once;
-            static os_log_t mapbox_log;
+            static os_log_t info_log;
+            static os_log_t debug_log;
+            static os_log_t error_log;
+            static os_log_t fault_log;
             static os_log_type_t log_types[] = { OS_LOG_TYPE_DEFAULT,
                                                     OS_LOG_TYPE_INFO,
                                                     OS_LOG_TYPE_DEBUG,
                                                     OS_LOG_TYPE_ERROR,
                                                     OS_LOG_TYPE_FAULT };
             dispatch_once(&once, ^ {
-                mapbox_log = os_log_create("com.mapbox.maps-ios-sdk", "SDK");
+                info_log = os_log_create("com.mapbox.maps", "INFO");
+                debug_log = os_log_create("com.mapbox.maps", "DEBUG");
+                error_log = os_log_create("com.mapbox.maps", "ERROR");
+                fault_log = os_log_create("com.mapbox.maps", "FAULT");
             });
+            
+            os_log_t mapbox_log;
+            switch (level) {
+                case MGLLoggingLevelInfo:
+                    mapbox_log = info_log;
+                    break;
+                case MGLLoggingLevelDebug:
+                    mapbox_log = debug_log;
+                    break;
+                case MGLLoggingLevelError:
+                    mapbox_log = error_log;
+                    break;
+                case MGLLoggingLevelFault:
+                    mapbox_log = fault_log;
+                    break;
+                case MGLLoggingLevelNone:
+                default:
+                    break;
+            }
+
             NSUInteger logTypesCount = sizeof(log_types) / sizeof(os_log_type_t);
```

Comment:
As asserts are not enabled in Release config, this can trigger an “Unused variable ‘logTypesCount’” warning.